Client/Ecosia/Migration/EcosiaImport.swift

```diff
@@ -22,7 +22,8 @@ final class EcosiaImport {
 
     struct Failure: Error {
         enum Code: Int {
-            case favourites = 911,
+            case shutdown = 900,
```

Comment:
I think we don't need another code as it's clearly the migration of history that fails, right?